install/streams.csv

```diff
@@ -2339,10 +2339,10 @@ WHZ,10,50,false,false,true,1993-03-22T15:45:00Z,2001-04-05T14:34:00Z
 WHZ,10,100,false,false,false,2001-04-07T21:53:00Z,9999-01-01T00:00:00Z
 WHZ,11,50,false,false,true,2001-04-08T10:20:00Z,2001-10-08T17:24:00Z
 WHZ,20,1,false,false,false,2001-04-05T00:00:00Z,2014-06-05T05:00:00Z
-WHZ,20,50,false,false,false,2014-06-05T05:00:01Z,2020-01-14T19:47:30Z
```

Comment:
It looks like there are triggered 50hz from 2001 to 2014-06-04. From random checking:

```
./list_miniseed.sh WHZ 2002 88 91 | grep BN* | sed -e 's/"//g' -e 's/Key//' -e 's/://'
miniseed/2002/2002.088/WHZ.NZ/2002.088.WHZ.20-BNE.NZ.D
  miniseed/2002/2002.088/WHZ.NZ/2002.088.WHZ.20-BNN.NZ.D
  miniseed/2002/2002.088/WHZ.NZ/2002.088.WHZ.20-BNZ.NZ.D
  miniseed/2002/2002.089/WHZ.NZ/2002.089.WHZ.20-BNE.NZ.D
  miniseed/2002/2002.089/WHZ.NZ/2002.089.WHZ.20-BNN.NZ.D
  miniseed/2002/2002.089/WHZ.NZ/2002.089.WHZ.20-BNZ.NZ.D
  miniseed/2002/2002.090/WHZ.NZ/2002.090.WHZ.20-BNE.NZ.D
  miniseed/2002/2002.090/WHZ.NZ/2002.090.WHZ.20-BNN.NZ.D
  miniseed/2002/2002.090/WHZ.NZ/2002.090.WHZ.20-BNZ.NZ.D
  miniseed/2002/2002.091/WHZ.NZ/2002.091.WHZ.20-BNE.NZ.D
  miniseed/2002/2002.091/WHZ.NZ/2002.091.WHZ.20-BNN.NZ.D
  miniseed/2002/2002.091/WHZ.NZ/2002.091.WHZ.20-BNZ.NZ.D
```

```
./list_miniseed.sh WHZ 2010 100 102 | grep BN* | sed -e 's/"//g' -e 's/Key//' -e 's/://'
miniseed/2010/2010.100/WHZ.NZ/2010.100.WHZ.20-BNZ.NZ.D
  miniseed/2010/2010.101/WHZ.NZ/2010.101.WHZ.20-BNZ.NZ.D
  miniseed/2010/2010.102/WHZ.NZ/2010.102.WHZ.20-BNZ.NZ.D
```

```
./list_miniseed.sh WHZ 2014 51 53 | grep BN* | sed -e 's/"//g' -e 's/Key//' -e 's/://'
miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.10-BHE.NZ.D
  miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.10-BHN.NZ.D
  miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.10-BHZ.NZ.D
  miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.20-BNE.NZ.D
  miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.20-BNN.NZ.D
  miniseed/2014/2014.051/WHZ.NZ/2014.051.WHZ.20-BNZ.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.10-BHE.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.10-BHN.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.10-BHZ.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.20-BNE.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.20-BNN.NZ.D
  miniseed/2014/2014.052/WHZ.NZ/2014.052.WHZ.20-BNZ.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.10-BHE.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.10-BHN.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.10-BHZ.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.20-BNE.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.20-BNN.NZ.D
  miniseed/2014/2014.053/WHZ.NZ/2014.053.WHZ.20-BNZ.NZ.D

```

Reply:
Kia ora @mnaguit !
yeap indeed this is awkyard - something like questionable metadata or data.
the original delat metadata shows only:  
`WHZ,20,1,false,false,false,2001-04-05T00:00:00Z,2014-06-05T05:00:00Z`
And 
`WHZ,20,50,false,false,false,2014-06-05T05:00:01Z,2020-01-14T19:47:30Z`


- No mention of Continuous BN WHZ stream (hence deemed i d guess **Experimental** for that station without much info :))  
I checked that:
-  the BN streams is consistent with other LN/HN recorded (same data source) (kinda) --same instrumentation assumed
- The ON/OFF _50 Hz tap_ switch of strong 20 WHZ BN  was effective from 2002 till 2008
- WHZ BN 50 Hz  actually continuous records Say from 2009 till 2020 s 
- WHZ seems to have been an experimental stationsto test  configuration wrt to its metadta history.

I will amend the PR adding a patch to  figure a "continuous" BN 50  from 2002

Cheers!